### PR TITLE
add example webcam class

### PR DESCRIPTION
**Pull request by aurora-chicken**
Before issuing a pull request, please see the contributing page.

### CODE REVIEW

**Review by IsaccBarker — CHANGES_REQUESTED** [review]
Sorry, gotta regect this one. The main reason is that you are defining this as an opmode. You should define this as a class so we can use it from opmodes. You can see the logging class or the collision detection class as an example :)

**Comment by aurora-chicken** [comment]
Ok thx for the feedback, I'll change it to a class so we can implement it in opmodes.

**Comment by IsaccBarker** [comment]
Thanks. Good job on the code other than that by the way :)

**Comment by IsaccBarker** [comment]
also, just out of curiosity, may I ask why you changed Gradle and Android SDK versions?

**Comment by aurora-chicken** [comment]
I made some changes and I think the reason that gradle and android SDK were changed was because android studio automatically changed them

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/WebcamInit.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
+package org.firstinspires.ftc.teamcode;
```
Comment (IsaccBarker) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/WebcamInit.java:1]: Only one thing; rename this class just to `webcam` and move it into its own directory. That way the source tree is cleaner, and it's much easier for you to maintain :D

**Review by IsaccBarker — CHANGES_REQUESTED** [review]

**Comment by aurora-chicken** [comment]
Ok I made those changes :D

**Comment by IsaccBarker** [comment]
awesome!

**Review by IsaccBarker — APPROVED** [review]
nice job!

**Pull request merged**